lib/api/src/modules/refs.ts

```diff
@@ -180,10 +183,14 @@ export const init: ModuleFn = ({ store, provider, fullAPI }) => {
               credentials,
               cache: 'no-cache',
             })
-          ),
+          ).then((r: SetRefData) => (r?.error?.message === 'Failed to fetch' ? {} : r)),
```

Comment:
I think we should only run this block if `included` is true.

Reply:
Also -- do we have tests for this stuff?

Reply:
I think in principle metadata.json is optional right?
So if the request errors for whatever reason, I think it's ok to ignore that error, rather then showing it to the user?

Reply:
It is fine to silently drop it if it errors, I agree. But given we pass credentials, and we already know going in if the credentials are going to work (based on `/stories.json`), then we can be a bit smarter no?

Or is it parallel? If so let's silently ignore errors.

Reply:
Yes it's partially in parallel.